src/settings.ts

```diff
@@ -1596,6 +1596,18 @@ export class RemotelySaveSettingTab extends PluginSettingTab {
           });
       });
 
+    new Setting(basicDiv)
+      .setName(t("settings_ignorepaths"))
+      .setDesc(t("settings_ignorepaths_desc"))
```

Comment:
I don't know if I'm doing it right, but if if I try to `npm run build` with your changes, I get:
```
ERROR in /home/sv-cheats-1/Documents/PROJECTS/remotely-save/src/settings.ts
./src/settings.ts 1600:17-39
[tsl] ERROR in /home/sv-cheats-1/Documents/PROJECTS/remotely-save/src/settings.ts(1600,18)
      TS2345: Argument of type '"settings_ignorepaths"' is not assignable to parameter of type '"confirm" | "disable" | "enable" | "goback" | 
"submit" | "sometext" | "syncrun_alreadyrunning" | "syncrun_syncingribbon" | "syncrun_step0" | "syncrun_step1" | "syncrun_step2" | ... 230 mor
e ... | "syncalgov2_button_disagree"'.
ts-loader-default_e3b0c44298fc1c14
 @ ./src/main.ts 13:0-52 415:35-57

ERROR in /home/sv-cheats-1/Documents/PROJECTS/remotely-save/src/settings.ts
./src/settings.ts 1601:17-44
[tsl] ERROR in /home/sv-cheats-1/Documents/PROJECTS/remotely-save/src/settings.ts(1601,18)
      TS2345: Argument of type '"settings_ignorepaths_desc"' is not assignable to parameter of type '"confirm" | "disable" | "enable" | "gobac
k" | "submit" | "sometext" | "syncrun_alreadyrunning" | "syncrun_syncingribbon" | "syncrun_step0" | "syncrun_step1" | "syncrun_step2" | ... 23
0 more ... | "syncalgov2_button_disagree"'.
ts-loader-default_e3b0c44298fc1c14
 @ ./src/main.ts 13:0-52 415:35-57
```
Seems like there's a type issue in `settings.ts` that does not seem to like the new `settings_ignorepaths` field